2-mandatory/1-recipes.js

```diff
@@ -22,4 +22,86 @@
   You should write and log at least 5 recipes
 */
 
-// write code here
\ No newline at end of file
+// write code here
+const myRecipes = {
+  Recipe1: {
+    Title: "Pho",
+    Serves: 1,
+    Ingredients: ["rice noodles", "broth", "beef", "herbs", "limes", "chilies"],
+  },
+  Recipe2: {
+    Title: "Satay",
+    Serves: 2,
+    Ingredients: [
+      "chicken",
+      "peanut sauce",
+      "chili sauce",
+      "shallots",
+      "sambal",
+      "limes juice",
+      "chiles",
+    ],
+  },
+  Recipe3: {
+    Title: "Bakso",
+    Serves: 2,
+    Ingredients: ["mince beef", "flour", "pepper", "eggs", "baking powder"],
+  },
+  Recipe4: {
+    Title: "Nasi Goreng ",
+    Serves: 1,
+    Ingredients: [
+      "shrimp paste",
+      "chilli",
+      "garlic",
+      "chicken",
+      "kecap manis",
+      "onion",
+      "cooked cold rice",
+    ],
+  },
+  Recipe5: {
+    Title: "Bubur Ayam",
+    Serves: 2,
+    Ingredients: [
+      "rice",
+      "chicken",
+      "fish sauce ",
+      "white pepper",
+      "green onions",
+    ],
+  },
+};
+// console.log(myRecipes);
+
+// Object.values(myRecipes.Recipe1).forEach((recipe) => console.log(recipe));
+
+// Object.entries(myRecipes.Recipe1).forEach(([key, value]) => {
+//   console.log(`${key}: ${value}`);
+// });
+
+// var myRecipesCount = Object.keys(myRecipes).length;
+// console.log(myRecipesCount);
+
+// function favoriteRecipe(recipeObj) {
+//   for ([key, value] of Object.entries(recipeObj)) {
+//     console.log(`${key}: ${value}`);
+//   }
+// }
+// favoriteRecipe(myRecipes.Recipe1);
+// favoriteRecipe(myRecipes.Recipe2);
+// favoriteRecipe(myRecipes.Recipe3);
+// favoriteRecipe(myRecipes.Recipe4);
+// favoriteRecipe(myRecipes.Recipe5);
+
+// for (const recipe in myRecipes) {
+//   for (const [key, value] of Object.entries(myRecipes[recipe])) {
+//     console.log(`${key}: ${value}`);
+//   }
+// }
+
+for (const recipe in myRecipes) {
+  for (const key in myRecipes[recipe]) {
+    console.log(`${key}: ${myRecipes[recipe][key]}`);
+  }
+}
```

Comment:
good job, you can also put receipts inside of the array (instead of objects) and use array functions to console.log keys :) can you think of it and write code for this?

Reply:
Hi @ellietms,
Thanks for the review.
Could you elaborate you task please?
What I don't undersatnd is do you want me to delcare all of my recepies inside and array then console.log them.
like this:
```
const myRecipes = [ {
  Recipe1: {
    Title: "Pho",
    Serves: 1,
    Ingredients: ["rice noodles", "broth", "beef", "herbs", "limes", "chilies"],
  },
  Recipe2: { ...
  ]
```
Then going through them and console.log each reipes in seperate array!?